lib/handler.js

```diff
@@ -46,4 +48,4 @@ async function handler(event) {
     }
 }
 
-module.exports = handler
+module.exports = handler, decorateWithIamToken
```

Comment:
i think this can just be
```suggestion
module.exports = handler
```

Reply:
indeed!